commands/ssh.go

```diff
@@ -70,14 +64,18 @@ func RunSSH(ns string, out io.Writer) error {
 		return errSSHInvalidOptions
 	}
 
+	// CoreOS has no root user
```

Comment:
CoreOS not having a root user is a usability issue that we can use the cli to alleviate. I feel strongly that the ssh command should handle these sorts of conditions for the user.